src/cf2tf/conversion/overrides.py

```diff
@@ -51,6 +51,11 @@ def tag_conversion(_tc: "TemplateConverter", params: CFParams) -> CFParams:
         return params
 
     original_tags: ListType = params["Tags"]  # type: ignore
+    # Handle Cloudformation templates that generate Tags: []
+    if not original_tags:
+        params["tags"] = params["Tags"]
+        del params["Tags"]
+        return params
```

Comment:
```python3
params["tags"] = params["Tags"]
```

If `original_tags` is `[]`, then I think we can just do the `del` and then return. Not sure why we would want to preserve the `[]` and move it from `Tags` to `tags`.